libs/chat-workflow/src/main/java/org/finos/springbot/workflow/java/resolvers/ChatButtonWorkflowResolverFactory.java

```diff
@@ -0,0 +1,46 @@
+package org.finos.springbot.workflow.java.resolvers;
+
+import java.util.Optional;
+
+import org.finos.springbot.workflow.annotations.ChatButton;
+import org.finos.springbot.workflow.java.mapping.ChatHandlerExecutor;
+
+/**
+ * Get chat button detail in controller 
```

Comment:
Can you describe what this is for?

Reply:
As we discussed last time, we need button information on controller method. For that we have added this class.